src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/Direction.java

```diff
@@ -13,6 +13,7 @@
     private String dateSent;
     private DirectionTag tag;
     private List<IdValue<PreviousDates>> previousDates;
+    private List<IdValue<ClarifyingQuestion>> clarifyingQuestions;
 
     private Direction() {
         // noop -- for deserializer
```

Comment:
Would this class be missing another constructor that includes clarifying clarifyingQuestions.